spec/features/result_spec.rb

```diff
@@ -0,0 +1,25 @@
+feature 'Displays result' do
+  before do
+    allow(Computer::WEAPONS).to receive(:sample).and_return(:scissors)
```

Comment:
Really like how you've set your tests up this way to check every outcome! I didn't realise this could be done in feature tests definitely something for me to remember for next time! (: